workflow/infrastructure-monitoring/container-monitoring.mdx

```diff
@@ -1,73 +1,131 @@
+        <Accordion title="Deployments">
+            Deployments provide declarative updates to applications. They are a high-level abstraction that simplifies the management and scaling of applications.
+        </Accordion>
 
-### Block IO service recursive
-The container block I/O usage refers to the amount of data that is read from or written to a block device, such as a disk or storage volume, by a container. this is typically measured in terms of the number of input/output (I/O) operations and the amount of data that is transferred during those operations. The container.blockio.io_service_bytes_recursive metric key provides information about the amount of data that is read from or written to the block device in bytes, broken down by individual operations.
+        <Accordion title="ReplicaSets">
+            ReplicaSets ensure a specified number of replica pods are running at any given time. ReplicaSets can be used for horizontally scaling your applications. 
+        </Accordion>
 
-![](/images/infra-monitoring/c5.png)
+        <Accordion title="Jobs">
+            Jobs are controller objects that represent finite and short-lived tasks or batch processes that run within pods.
+        </Accordion>
+
+        <Accordion title="CronJobs">
+            CronJobs are controller objects that can be scheduled to automate task execution. CronJobs are Jobs that can be run on a periodic schedule.
+        </Accordion>
+
+        <Accordion title="DaemonSet">
+            DaemonSets are controllers that ensure a specific set of pods is running across your cluster. DaemonSets are used to run exactly one instance of a pod on each node in the cluster.
+        </Accordion>
+
+        <Accordion title="StatefulSet">
+            StatefulSets are used for managing stateful applications that are comprised of unique network identifiers, persistent storage, and specific ordering requirements for their deployment and scaling. 
+        </Accordion>
+
+        </AccordionGroup>
+
+</Accordion>
+
+# Usage Graph
+
+The Usage graph displays all of your team's Infrastructure usage data. Gain immediate insight into your stack's performance.
+
+### Change Usage Type
+
+Toggle between your Infrastructure CPU and Memory usage over time with the Usage Graph.
+
+<img src="https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExaDRkZHlldGN3bWJ4b281emh3Ym52ZHFpaGJ4anF2OWtzZTMyMW9vbiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/CMxUd2XqN2sQKFTXEW/giphy.gif" width="750" height="750"/>
+
+### Select Date Range
+
+Select your own date range from the calendar or choose from our default options.
+
+<img src="https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExMWpuamp5amFsM3dtazFobXVibnhpb2xmc3NqOWN6d3djczM4anQydCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/9ZT90BmYhPrqB3fiWt/giphy.gif" width="750" height="750"/>
+
+# Data Table
+
+The List Tab displays all of your specific Infrastructure data in a single table. You can manage your list view by adding and removing specific metrics.
```

Comment:
Same comment as above in the Usage Graph section. Take a stab at editing this to summarize below sections in one or two sentences.